### ISSUE TITLE
`sensible-editor: Editing failed` in CI - that's expected

### ISSUE THREAD

**Issue by TomasTomecek** [thoth-station/thamos#1131]
## Bug description
We are running thamos in CI via yours pre-commit hook and getting this output:
```
/usr/bin/sensible-editor: 25: editor: not found
/usr/bin/sensible-editor: 28: nano: not found
/usr/bin/sensible-editor: 31: nano-tiny: not found
/usr/bin/sensible-editor: 34: vi: not found
Couldn't find an editor!
Set the $EDITOR environment variable to your desired editor.
2022-06-30 13:28:37,371 [123] ERROR    thamos: sensible-editor: Editing failed
              Runtime environment and configuration check results
```

Since this is running in CI, it's expected that `$EDITOR` is not available. Even pty may not be available. I'd suggest doing `$editor` magic only when it's needed.

Probably having an option for CI runs would be perfect.

### Expected behavior
no such logs as described above

### Environment information
pre-commit CI (probably ubuntu)

### Additional context
https://github.com/packit/ogr/pull/722
https://results.pre-commit.ci/run/github/161636700/1656595682.f1bfqPiYRaafQs0E09kC4g

**Comment by mayaCostantini** [comment]
Yes, this will be fixed by https://github.com/thoth-station/thoth-pre-commit-hook/pull/27 :)

**Comment by codificat** [comment]
> Yes, this will be fixed by [thoth-station/thoth-pre-commit-hook#27](https://github.com/thoth-station/thoth-pre-commit-hook/pull/27) :)

which is now merged, so
/close

thanks for the report and the fix!
